src/controllers/userController.js

```diff
@@ -202,9 +204,91 @@ userController.post('/reset-password', async (req, res) => {
     }
 });
 
+//Approving/ Unapproving user
+userController.put('/approve/:id', async (req, res) => {
+    try {
+        const userId = req.params.id;
+        const isAdmin = req.requester.role == 'admin';
```

Comment:
but what if admin and isDeleted === true ?

Reply:
fixed for not deleted.

Reply:
throw new Error('You do not have rights to modify the record!');

Reply:
Where is the fix?

Reply:
The fix fore isDeleted == true is in the userService:
    if (existingUser.isDeleted == true) {
        throw new Error('User is deleted!');
    }
    Not sure about :
![image](https://github.com/Getsov/Motorsport-Events-Server/assets/93388225/2c46ec77-2b7e-46bc-86e2-acf7e7e3e05d)
the part with admin

Reply:
fixed in userSerice